### PR TITLE
show upgrade policy in clusters view

### PR DESCRIPTION
**Pull request by jmelis**

### CODE REVIEW

File: src/pages/elements/Clusters.js [hunk 1]
```diff
@@ -1,61 +1,185 @@
...
+            formatters: [headerFormat]
+          },
+          cell: {
+            formatters: [cellFormat]
+          },
+          property: 'channel'
+        },
+        {
+          header: {
+            label: 'External ID',
```
Comment (maorfr) [inline src/pages/elements/Clusters.js:74]: i'd use id over external id, as that is the language we speak with OCM/SREP

File: src/pages/elements/Clusters.js [hunk 2]
```diff
@@ -1,61 +1,185 @@
...
+                </>
+              ),
+              cellFormat
+            ]
+          },
+          property: 'name_path'
+        },
+        {
+          header: {
+            label: 'Description',
```
Comment (maorfr) [inline src/pages/elements/Clusters.js:44]: i'd even consider removing the description from this view?

**Review by maorfr — APPROVED** [review]
LGTM

great consolidation. i like it that we understand that clusters are not pets anymore :)

File: src/pages/elements/Clusters.js (hunk 2 above)
Comment (jmelis) [reply src/pages/elements/Clusters.js:44]: I think I'm going to leave it for now

File: src/pages/elements/Clusters.js (hunk 1 above)
Comment (jmelis) [reply src/pages/elements/Clusters.js:74]: good feedback, thanks!

**Pull request merged**